risc0/zkp/Cargo.toml

```diff
@@ -29,7 +29,7 @@ serde = { version = "1.0", default-features = false, features = ["derive"] }
 tracing = { version = "0.1", default-features = false, features = [
   "attributes",
 ] }
-
+libm = "0.2"
```

Comment:
This might cause issues for guest and no-std builds. I'm thinking we can make this dep optional and enable with the `soundness` flag.